.github/workflows/update-manifest.yaml

```diff
@@ -0,0 +1,52 @@
+name: Update manifests
+
+on:
+  workflow_dispatch:
+    # Enable manual trigger of this action.
+    inputs:
+      api-manager-version:
+        description: api manager version
+        default: ""
+        required: false
+      portal-manager-version:
+        description: portal manager version
+        default: ""
+        required: false
+      node-manager-version:
+        description: portal manager version
+        default: ""
+        required: false
+
+
+jobs:
+  update-manifests:
+    runs-on: ubuntu-latest
+    name: Update manifests
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.ACTION_TOKEN }}
+      - name: Generate manifests
+        run: |
+         if  [[ -n "${{ github.event.client_payload.api-manager-version }}" ]]; then
+            sed -i 's/API_MANAGER_VERSION ?= v.*/API_MANAGER_VERSION ?= ${{ github.event.client_payload.api-manager-version }}/' Makefile
+            make api-manager
+         fi
+         if [[ -n "${{ github.event.client_payload.portal-manager-version }}" ]]; then
+            sed -i 's/PORTAL_MANAGER_VERSION ?= v.*/PORTAL_MANAGER_VERSION ?= ${{ github.event.client_payload.portal-manager-version }}/' Makefile
+            make portal-manager
+         fi
+         if [[ -n "${{ github.event.client_payload.node-manager-version }}" ]]; then
+            sed -i 's/NODE_MANAGER_VERSION ?= v.*/NODE_MANAGER_VERSION ?= ${{ github.event.client_payload.node-manager-version }}/' Makefile
+            make node-manager
+         fi
+      - name: Create Pull Request
```

Comment:
This nice, and i like it, but i suggest to run this only on PR builds. I don't want to run this flow when we run image build and publish in the same time. So image builds must happen only and if manifests are up to date. I suggest a validation before image build starts.

Reply:
My initial plan was to make a release of api-manager trigger this pipeline. 

It shouldn't run on push or PR.

@mhmxs how does that sound?